packages/react-components/react-components/package.json

```diff
@@ -21,7 +21,7 @@
     "start": "yarn storybook",
     "docs": "api-extractor run --config=config/api-extractor.local.json --local",
     "build:local": "tsc -p ./tsconfig.lib.json --module esnext --emitDeclarationOnly && node ../../../scripts/typescript/normalize-import --output ./dist/types/packages/react-components/react-components/src && yarn docs",
-    "storybook": "node ../../../scripts/storybook/runner --port 3000 --no-manager-cache",
```

Comment:
Is it expected to remove `--port` & `--no-manager-cache` there?

Reply:
good catch! in general this is not "used" anymore and will be normalized in some follow-up (we have now public-docsite-v9) to gather all stories etc